app/controllers/nonsipp/membertransferout/RemoveTransferOutController.scala

```diff
@@ -0,0 +1,135 @@
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.SaveService
+import viewmodels.DisplayMessage.Message
+import viewmodels.implicits._
+import viewmodels.models.{FormPageViewModel, YesNoPageViewModel}
+import views.html.YesNoPageView
+
+import javax.inject.{Inject, Named}
+import scala.concurrent.{ExecutionContext, Future}
+
+class RemoveTransferOutController @Inject()(
+  override val messagesApi: MessagesApi,
+  @Named("non-sipp") navigator: Navigator,
+  identifyAndRequireData: IdentifyAndRequireData,
+  formProvider: YesNoPageFormProvider,
+  saveService: SaveService,
+  val controllerComponents: MessagesControllerComponents,
+  view: YesNoPageView
+)(implicit ec: ExecutionContext)
+    extends PSRController
+    with I18nSupport {
+
+  private val form = RemoveTransferOutController.form(formProvider)
+
+  def onPageLoad(srn: Srn, memberIndex: Max300, index: Max5): Action[AnyContent] =
+    identifyAndRequireData(srn) { implicit request =>
+      val nameDOB = request.userAnswers.get(MemberDetailsPage(srn, memberIndex)).get
+      val receivingSchemeName = request.userAnswers.get(ReceivingSchemeNamePage(srn, memberIndex, index))
+      receivingSchemeName match {
+        case Some(schemeName) =>
+          Ok(
+            view(
+              form,
+              RemoveTransferOutController
+                .viewModel(srn, memberIndex: Max300, index: Max5, nameDOB.fullName, schemeName)
+            )
+          )
+        case None =>
+          Redirect(
+            controllers.nonsipp.membertransferout.routes.SchemeTransferOutController.onPageLoad(srn, NormalMode).url
+          )
+      }
+    }
+
+  def onSubmit(srn: Srn, memberIndex: Max300, index: Max5): Action[AnyContent] =
+    identifyAndRequireData(srn).async { implicit request =>
+      form
+        .bindFromRequest()
+        .fold(
+          formWithErrors => {
+            (
+              for {
+                nameDOB <- request.userAnswers.get(MemberDetailsPage(srn, memberIndex)).getOrRecoverJourneyT
+                receivingSchemeName <- request.userAnswers
+                  .get(ReceivingSchemeNamePage(srn, memberIndex, index))
+                  .getOrRecoverJourneyT
+              } yield BadRequest(
+                view(
+                  formWithErrors,
+                  RemoveTransferOutController.viewModel(srn, memberIndex, index, nameDOB.fullName, receivingSchemeName)
+                )
+              )
+            ).merge
+          },
+          removeDetails => {
+            if (removeDetails) {
+              for {
+                updatedAnswers <- Future
+                  .fromTry(request.userAnswers.remove(transferOutPages(srn, memberIndex, index)))
+                _ <- saveService.save(updatedAnswers)
+              } yield Redirect(
```

Comment:
Both transfersin and out are missing submission here. Reference: RemoveEmployerContributionsController
But don't do it now @ccatosdevelopment , we can cover this when we are doing the BE tickets.